src/common/src/util/mod.rs

```diff
@@ -22,14 +22,14 @@ pub mod deployment;
 pub mod env_var;
 pub mod epoch;
 pub mod hash_util;
-pub mod iter_util;
+pub use rw_iter_util as iter_util;
 pub mod memcmp_encoding;
 pub mod meta_addr;
 pub mod panic;
 pub mod pretty_bytes;
 pub mod prost;
 pub mod query_log;
-pub mod resource_util;
+pub use rw_resource_util as resource_util;
```

Comment:
Why not changing all the usage? IMO later if we have two ways to import the same module, things can be messy

Reply:
> Why not changing all the usage? 

Because it's widely used in many places. This is kind of "prelude" style. And I now think it's both OK whether or not to change it (but I'm lazy).

> IMO later if we have two ways to import the same module, things can be messy

I don't think this may cause any trouble. 👀

Reply:
Just perfectionism🤡